bundles/net.tourbook.export/src/net/tourbook/export/fit/FitExporter.java

```diff
@@ -0,0 +1,357 @@
+/*******************************************************************************
+ * Copyright (C) 2023 Frédéric Bard and Contributors
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.export.fit;
+
+import com.garmin.fit.ActivityMesg;
+import com.garmin.fit.DateTime;
+import com.garmin.fit.DeveloperDataIdMesg;
+import com.garmin.fit.DeviceIndex;
+import com.garmin.fit.DeviceInfoMesg;
+import com.garmin.fit.Event;
+import com.garmin.fit.EventMesg;
+import com.garmin.fit.EventType;
+import com.garmin.fit.File;
+import com.garmin.fit.FileEncoder;
+import com.garmin.fit.FileIdMesg;
+import com.garmin.fit.Fit;
+import com.garmin.fit.FitRuntimeException;
+import com.garmin.fit.Gender;
+import com.garmin.fit.LapMesg;
+import com.garmin.fit.Manufacturer;
+import com.garmin.fit.Mesg;
+import com.garmin.fit.RecordMesg;
+import com.garmin.fit.SessionMesg;
+import com.garmin.fit.Sport;
+import com.garmin.fit.UserProfileMesg;
+import com.garmin.fit.util.SemicirclesConverter;
+
+import java.nio.ByteBuffer;
+import java.time.Instant;
+import java.time.Period;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import net.tourbook.application.TourbookPlugin;
+import net.tourbook.common.UI;
+import net.tourbook.common.time.TimeTools;
+import net.tourbook.common.util.StatusUtil;
+import net.tourbook.data.TourData;
+import net.tourbook.data.TourMarker;
+import net.tourbook.data.TourPerson;
+import net.tourbook.export.Activator;
+
+import org.osgi.framework.Version;
+
+public class FitExporter {
```

Comment:
Gear ratio is not exported

![image](https://user-images.githubusercontent.com/1283445/232201193-433bf769-b0bf-4c84-a02d-7c143f295c62.png)

Reply:
Heartbeat variability is not exported

![image](https://user-images.githubusercontent.com/1283445/232215326-ccf9cd1b-e202-42e1-a8d3-839fd5d8823f.png)

![image](https://user-images.githubusercontent.com/1283445/232215393-e0a515a8-216c-4b97-9c3b-8953668af4ba.png)

Reply:
I don't have gear ratio so I may have missed that.
I will add gear ratio and HRV to the FIT export

Reply:
> Gear ratio is not exported
> 
I have fixed this but was not able to thoroughly check since I don't have any files containing gear data

Reply:
> Heartbeat variability is not exported
> 
I have fixed it and HRV is now exported